Subject: remove absolte patch from comment
From: olh@suse.de

autobuild doesnt like the $RPM_BUILD_ROOT string being there

 * This file was generated by /var/tmp/kernel-source-2.6.14_rc4-build/usr/src/linux-2.6.14-rc4-2/Kbuild


 Kbuild |    2 --
 1 files changed, 2 deletions(-)

diff -r 2df05bb69f39 Kbuild
--- a/Kbuild	Mon Mar  6 12:31:34 2006 -0500
+++ b/Kbuild	Mon Mar  6 12:32:45 2006 -0500
@@ -28,8 +28,6 @@ define cmd_offsets
 	 echo "/*"; \
 	 echo " * DO NOT MODIFY."; \
 	 echo " *"; \
-	 echo " * This file was generated by $(srctree)/Kbuild"; \
-	 echo " *"; \
 	 echo " */"; \
 	 echo ""; \
 	 sed -ne $(sed-y) $<; \
